aidants_connect_web/models.py

```diff
@@ -492,3 +492,9 @@ def save(self, *args, **kwargs):
 
     def delete(self, *args, **kwargs):
         raise NotImplementedError("Deleting is not allowed on journal entries")
+
+    @property
+    def usager_id(self):
+        if self.usager:
+            return int(self.usager.split(" - ")[1])
+        return None
```

Comment:
J'espère qu'on va pouvoir rajouter bientôt une vraie FK dans la table 😅 

D'ici là, pour un peu plus de robustesse / résilience, je propose
```suggestion
        try:
            return int(self.usager.split(" - ")[1])
        except (IndexError, ValueError):
            return None
```